lib/clients/jira.go

```diff
@@ -349,7 +352,12 @@ func (j realJIRAClient) request(f func() (interface{}, *jira.Response, error)) (
 	b := backoff.NewExponentialBackOff()
 	b.MaxElapsedTime = j.config.GetTimeout()
 
-	er := backoff.Retry(op, b)
+	er := backoff.RetryNotify(op, b, func(err error, duration time.Duration) {
```

Comment:
nit: it's functionally the same, but I think it makes more sense to use `err` in the outer scope and `er` in the inline func.